### ISSUE TITLE
[TW-1764] When deleting recurring task all tasks, including completed tasks, are marked as deleted.

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1788]
_Alan Young on 2016-02-01T00:51:06Z says:_ 

 When deleting a recurring task and 'yes' is selected for deleting all pending tasks, then all tasks--including completed tasks--are marked as deleted.

Completed tasks should not be included when marking as deleted, only pending and waiting recurring tasks.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2016-02-01T00:51:06Z
Modified: 2017-01-16T18:37:59Z
```

**Comment by tbabej** [comment]
Fixed on 2.6.0.
